observability/metrics/src/main/scala/com/daml/metrics/ExecutorServiceMetrics.scala

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import java.util.concurrent.{ExecutorService, ForkJoinPool, ThreadPoolExecutor}
+
+import com.daml.metrics.api.MetricHandle.Factory
+import com.daml.metrics.api.{MetricName, MetricsContext}
+import org.slf4j.LoggerFactory
+
+class ExecutorServiceMetrics(factory: Factory) {
+
```

Comment:
Can we possibly add at least happy path tests to check that these instrumentation methods work, please?